mlrun/api/api/endpoints/workflows.py

```diff
@@ -220,7 +220,7 @@ def _is_requested_schedule(
         return workflow_spec.schedule is not None
 
     project_workflow = _get_workflow_by_name(project, name)
-    return bool(project_workflow.get("schedule"))
+    return bool(project_workflow.get("schedule")) if project_workflow else False
```

Comment:
Did you run into a situation where this exploded? I mean, in that case there is no `workflow_spec` and also the project does not contain the workflow, so what are we running here? Seems to me that such a scenario should be failed anyway.

Reply:
it is a bit defensive to when u try to manually submit with empty spec and workflow does not exist.